### PR TITLE
Makes normal syndicate hardsuits fully acid proof

### PR DESCRIPTION
**Pull request by Therealdoooc213**
## About The Pull Request

in the title

## Why It's Good For The Game

A chemist with 50 units of flourosulfuric acid completely shutting down a nuclear operative isnt fun for anyone except the chemist.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

https://github.com/user-attachments/assets/40919bfa-1241-46fd-9eea-a2c91ffd1e80

</details>

## Changelog
:cl:
balance: syndicate hardsuits are fully acid proof now
/:cl:

### CODE REVIEW

**Comment by Therealdoooc213** [comment]
oops, i accidentally pressed enter before finishing it

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -535,8 +535,9 @@
 	item_state = "syndie_hardsuit"
 	hardsuit_type = "syndi"
 	w_class = WEIGHT_CLASS_NORMAL
+	resistance_flags = ACID_PROOF
 	supports_variations = DIGITIGRADE_VARIATION
-	armor = list(MELEE = 40,  BULLET = 50, LASER = 30, ENERGY = 55, BOMB = 35, BIO = 100, RAD = 50, FIRE = 50, ACID = 90, STAMINA = 60, BLEED = 70)
+	armor = list(MELEE = 40,  BULLET = 50, LASER = 30, ENERGY = 55, BOMB = 35, BIO = 100, RAD = 50, FIRE = 50, ACID = 100, STAMINA = 60, BLEED = 70)
```
Comment (XeonMations) [inline code/modules/clothing/spacesuits/hardsuit.dm:540]: Unsure if required, but i'd personally also make the helmets acid proof. JUST in case.

**Comment by MarkusLarsson421** [comment]
I'd make it like 70% acid resistant, being fully acid resistant feels weird. Either way, it wouldn't make them weirdly immune to acid for no explicit reason other than "balance".

**Comment by Therealdoooc213** [comment]
> I'd make it like 70% acid resistant, being fully acid resistant feels weird. Either way, it wouldn't make them weirdly immune to acid for no explicit reason other than "balance".

Nukies shouldnt be instantly shut down by a easily available chemical and be forced to buy elite hardsuits to avoid it.

**Comment by MarkusLarsson421** [comment]
Would 70% acid resistance easily shut down nukies?

**Comment by Therealdoooc213** [comment]
> Would 70% acid resistance instantly shut down nukies?
> 
> Try throwing a 100u beaker and a 300u beaker at someone with nukie armour. 300 units might be reasonable, but not 100u as it is very easily accessible.

It would. Nukies used to have 90 acid protection on the normal hardsuit and the hardsuit was still almost instantly destroyed with 50 units. 70% acid protection is useless.

**Review by PowerfulBacon — APPROVED** [review]
Avoid just needs a general refactor to be less annoying tbh

**Pull request merged**